### ISSUE TITLE
No option of main invocation

### ISSUE THREAD

**Issue by sach999** [actions-on-google/actions-builder-codelab-level-1#7]
There is no option of main invocation in actions console from where we can add a custom welcome message

**Comment by Fleker** [comment]
While you may be able to use a webhook, you should also see a **Send prompts** option that will let you add a basic message within the console directly.

![image](https://user-images.githubusercontent.com/3291635/134038691-796cb7dd-312f-4f59-990c-0ef0b1d1d0ca.png)

**Comment by sach999** [comment]
![image](https://user-images.githubusercontent.com/43383547/135617000-536748c3-5549-4921-98f9-bfb13acaf8fc.png)
This is the page I am getting with no option to select Main Invocation

**Comment by Fleker** [comment]
Are you using Dialogflow? What do you see in the *Actions* tab?

**Comment by sach999** [comment]
![image](https://user-images.githubusercontent.com/43383547/135657954-d277cdb2-3ee7-44f1-8136-8c9cc3f5fbd0.png)
This is what I see right now. When I click on 'Get Started' and click on build custom intent...it takes me to dialogflow. Is this because I've used dialogflow before starting this codelab?

**Comment by Fleker** [comment]
Yeah if your project is using Dialogflow, then this codelab will not work as intended. You'll need to convert the project to use Actions Builder or use a new project.

**Comment by lesliepound** [comment]
1. I am getting the same problem. It says " Invoke Main invocation"  in the documentation, but my screen does not say that or allow it. 
2.  Followed the **exact steps** from here: "Build Actions for Google Assistant using Actions Builder (Level 1)" ..  to get to this point.
3. Please change the documentation and advice with links to new documentation.

**Comment by Fleker** [comment]
You should create a new project to complete this codelab.

**Comment by peterdresslar** [comment]
I ran into the same issue reported by the initial poster. Then deleted project and reran the tutorial steps. I think there is actually something that can go wrong during new Actions project provisioning that causes this--there was a pretty big pause and an error message the first time I created a project.
